clients/unshaded/src/main/java/tachyon/client/file/options/OutStreamOptions.java

```diff
@@ -46,6 +46,19 @@ public Builder(TachyonConf conf) {
     }
 
     /**
+     * Creates a new builder from {@link OutStreamOptions}.
+     *
+     * @param options the options
+     */
+    public Builder(OutStreamOptions options) {
+      mBlockSize = options.getBlockSize();
+      mHostname = options.getHostname();
+      mTachyonStorageType = options.getTachyonStorageType();
+      mTTL = options.getTTL();
+      mUnderStorageType = options.getUnderStorageType();
+    }
```

Comment:
hmm, I think it's natural to put it here, it's just a a copy() method, right? If it's in util, which util class should it go to?